src/Context/HelperContext.php

```diff
@@ -83,6 +87,13 @@ class HelperContext extends RawDrupalContext implements SnippetAcceptingContext
    */
   public $currentScenario;
 
+  /**
+   * The current Behat scenario name.
+   *
+   * @var string
+   */
+  public $scenarioName;
```

Comment:
Discouraged to keep public props. Better if you provide a public getter method instead.